functions/exercise14/script.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+function getAmountOfCoins(amount) {
+    var denominationsOfCoins = [25, 10, 5, 2, 1];
+    var amountOfCoins = [];
+    for (var i = 0; i < denominationsOfCoins.length; i++) {
+        if (amount >= denominationsOfCoins[i]) {
```

Comment:
You refer 3 times to denominationsOfCoins[i]. You should remember this value in a variable and refer to this variable to make it quicker.